target/linux/mpc85xx/image/Makefile

```diff
@@ -1,107 +1,92 @@
 #
-# Copyright (C) 2010 OpenWrt.org
-#
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/image.mk
 
-define imgname
-$(BIN_DIR)/$(IMG_PREFIX)-$(2)-$(patsubst jffs2-%,jffs2,$(patsubst squashfs-%,squashfs,$(1)))
-endef
-
-define sysupname
-$(call imgname,$(1),$(2))-sysupgrade.bin
+DEVICE_VARS += DEVICE_PROFILE DTB_SIZE
+
+define Device/Default
+  PROFILES := Default
+  KERNEL_DEPENDS = $$(wildcard $(DTS_DIR)/$$(DEVICE_DTS).dts)
+  DEVICE_PROFILE :=
+  DEVICE_DTS :=
+  DTB_SIZE := 128k
+  KERNEL_ENTRY := 0x00000000
+  KERNEL_LOADADDR := 0x00000000
+  KERNEL := dtb | kernel-bin | gzip | uImage gzip
 endef
 
-define factoryname
-$(call imgname,$(1),$(2))-factory.bin
-endef
-
-zImage:=$(BIN_DIR)/$(IMG_PREFIX)-zImage
-
-DTS_TARGETS = hiveap-330 fsl/p1010rdb-pa tl-wdr4900-v1 fsl/p1020rdb
```

Comment:
While this may look like I removed support for p1010rdb-pa and p1020rdb, these boards were never actually supported within the code for this target besides the dtb generation. Thus, they were not moved over to the new style.